fibonacci/fibonacci.js

```diff
@@ -4,9 +4,8 @@ const fibonacci = function(count) {
   let a = 0;
   let b = 1;
   for (let i = 1; i < count; i++) {
-    const temp = b;
-    b = a + b;
-    a = temp;
+    // A slightly shorter version of the above solution:
+    // [a, b] = [b, a + b];
```

Comment:
Looks good! We just need to add back in the current solution:

```suggestion
    const temp = b;
    b = a + b;
    a = temp;
    // A slightly shorter version of the above solution:
    // [a, b] = [b, a + b];
```